### DISCUSSION TITLE
Buongiorno e qualche info generale su homebridge e homeassistan

### DISCUSSION THREAD

**Discussion by Curadecoco** [indomus/forum#1875] [GENERALE - Discussioni generiche]
Buongiorno a tutti, intanto complimenti per le guide egli articoli.
Mi sono d apo avvicinato a questo mondo con un piccolo sistema per le tende da esterno: ho usato i comandi Vimar con i gateway dedicato, poi ho caricato il tutto su Homekit di Apple perchè ho un tablet sempre a casa e mi va bene di usare il sistema della mela pe ril controllo.
Adesso volevo espandere la cosa, con qualche sensore di temperatura per la soffitta, l'attivazione del clima e anche controllo di vento e pioggia sempre le tende.
Vedo che soprattutto per gli ultimi due, è sempre necessario una stazione meteo che personalmente non mi serve, ma arrivando al dunque praticamente quasi tutti gli elementi hanno necessità di un gateway proprietario (questo senza considerare poi la compatibilità con homekit).
Volevo capire se tramite home bridge o home assistant io posso eliminare fisicamente il gateway di ogni marca per far convogliare sensori e sistemi in unico calderone; sto provando home assistant dal pc ma vedo che gli applicativi in realtà servano ad assumere il gateway.
In definitiva non sto capendo se sono costretto a munirmi di infiniti gateway per ogni marca e sistema poi da far convogliare.
La stessa domanda è ovviamente rivolta a home bridge che dovrebbe fare da ponte con il sistema della mela se il prodotto non è supportato dalla stessa; anche qui fa da ponte e basta o sostituisce in toto il gateway?
Esempio pratico: vorrei assimilare il sensore vento e pioggia della netatmo: devo prendermi la stazione lo stesso o basta Home assistant?  Quest'ultimo non mi ha fatto altro che far creare un account e lavora dal cloud i netatmo...
Grazi e a tutti per l'aiuto.

**Comment by snakuzzo** [comment]
Innanzi tutto, sulla questione HA o Homebridge, ti farei escludere a priori Homebridge (che di fatto è un bridge) in favore di HA (che è un vero e proprio hub) che ti permette di fare tante altre cose che nativamente con homekit (e con Homebridge) non puoi fare.
Tornando al discorso dei gateway ...ti dico che "dipende". Ci sono ad esempio device zigbee che magari puoi NON usare con ogni specifico gateway, ma utilizzare con soluzioni tipo Zigbee2mqtt o ZHA (direttamente su HA), dove di fatto utilizzi un solo coordinator per n device di marche diverse...il tutto per poi avere entità e device su HA da esporre a loro volta verso Homekit con l'omonima integrazione.
Altro discorso va fatto con device che necessitano per forza del loro bridge fisico, come ad esesempio Tadoº. In questo caso puoi integrare il loro bridge su HA sia via Cloud tramite integrazione nativa, che localmente sfruttando l'integrazione homekit controller (da non confondere con l'integrazione homekit).
Ovvio però che è un discorso che va fatto in base al device che scegli di utilizzare...i casi sono tanti.

**Reply by Scialla** [reply]
aggiungo che  il sensore vento e pioggia della netatmo necessita della stazione netatmo...

**Comment by Curadecoco** [comment]
Scusate il ritardo; grazie dell'aiuto.
Essendo neofita alcune cose non le ho capite bene.
Arriviamo ad un esempio pratico: vorrei installare un sensore pioggia, ed uno vento affiche poi interagiscano con le automazioni per le tende.
Per non comprare stazioni complete cosa mi suggerite?
